Manipulating derivatives example

EvanHughes2023 opened the pull request:
# New content

<!--- Briefly describe the changes made. -->

<!--- Replace XXX with any associated issues. --->

Resolves #32

## Checklist

- [ ] New pages are added to the table of contents.
- [ ] Everything displays correctly in the web-generated preview.
- [ ] Appropriate attribution has been given to source materials.
- [ ] My name is included alphabetically in `credits.md`.

mphoward commented on multivariable-calculus/manipulating-derivatives.md:
```diff
@@ -1 +1,67 @@
-# Manipulating partial derivatives
+# Manipulating Partial Derivatives
+
+## Example:
```
We can wrap all this in an "example" environment so it shows up like the example here:
https://mphowardlab.github.io/essential-math/linear-algebra/matrices.html

```suggestion
````{example} Change in internal energy with temperature and pressure
```

You will need to put quadruple ticks at the end of the example to close this block. You can check `linear-algebra/matrices.md` for an example.

mphoward commented on multivariable-calculus/manipulating-derivatives.md:
```diff
@@ -1 +1,67 @@
+
+We want to compute $\Delta U$ as a function of $T$ and $P$, so we need to determine:
+
+$$
+dU = \left( \frac{\partial U}{\partial T} \right)_P dT + \left( \frac{\partial U}{\partial P} \right)_T dP
+$$
```
A few suggestions for all equations:
* Please change the `$$` to `\begin{equation}` and `\end{equation}` so that they get numbered.
* Use `\d{}` for differentials
* It is optional if you'd like to change this since you have already written it correctly, but this book has a shortcut command `\td` for writing thermodynamic derivatives, so you could do like `\td{}{U}{T}{P}` and get the same as what you have for the first temperature derivative.
```suggestion
\begin{equation}
\d{U} = \left( \frac{\partial U}{\partial T} \right)_P \d{T} + \left( \frac{\partial U}{\partial P} \right)_T \d{P}
\end{equation}
```

mphoward commented on multivariable-calculus/manipulating-derivatives.md:
```diff
@@ -1 +1,67 @@
+
+***
+
+Given that:
+
+$$
+\alpha_V = \frac{1}{V} \left( \frac{\partial V}{\partial T} \right)_P
+$$
+
+$$
+\kappa_T = -\frac{1}{V} \left( \frac{\partial V}{\partial P} \right)_T
+$$
+
+$$
+c_P = \left( \frac{\partial H}{\partial T} \right)_P
+$$
+
+$$
+dU = T dS - P dV
+$$
+
+$$
+dH = T dS + V dP
+$$
+
+$$
+dG = -S dT + V dP
+$$
```
You can group these equations together using "align". Let's also try to define what the variables mean and add a little more text
```suggestion
Given that the thermal expansion coefficient $\alpha_V$, the isothermal compressibility $\kappa_T$, and the constant-pressure heat capacity $c_P$ are defined as:

\begin{align}
\alpha_V &= \frac{1}{V} \left( \frac{\partial V}{\partial T} \right)_P \\
\kappa_T &= -\frac{1}{V} \left( \frac{\partial V}{\partial P} \right)_T \\
c_P &= \left( \frac{\partial H}{\partial T} \right)_P,
\end{align}

where $V$ is the volume and $H$ is the enthalpy and that the following total differentials are known:

\begin{align}
\d{U} &= T \d{S} - P \d{V} \\
\d{H} &= T \d{S} + V \d{P} \\
\d{G} &= -S \d{T} + V \d{P}
\end{align}

where $S$ is the entropy and $G$ is the Gibbs free energy.
```

mphoward commented on multivariable-calculus/manipulating-derivatives.md:
```diff
@@ -1 +1,67 @@
+
+Form the derivatives using the given $dU$, then replace what we don't like:
+
+$$
```
Please use the align environment (like shown above) to break some of these equations into multiple lines at the equals signs. That will make it easier to read on smaller format screens.

mphoward commented on multivariable-calculus/manipulating-derivatives.md:
```diff
@@ -1 +1,67 @@
-# Manipulating partial derivatives
+# Manipulating Partial Derivatives
```
Please revert this change
```suggestion
# Manipulating partial derivatives
```

mphoward reviewed (commented):
Thank you for your work on this! This looks really great. Can you please apply the changes below, then Joshua & I will see if there is anything small needed to finalize and merge.

EvanHughes2023 commented:
Thanks so much for your feedback, Dr. Howard! GitHub’s interface and the MyST and LaTeX syntax definitely have a learning curve. I think I addressed all of your suggestions, but I’m having some trouble making the last few equations align properly. I did some more research and I got it to look better, but do you think it needs more work?

mphoward reviewed (approved):
Thank you for your contribution! This is great work. I made some edits to clean up the formatting and smooth the flow of the text. Please take a look at the final version if you decide you'd like to do another contribution.
